main/src/ui/notifications.vala

```diff
@@ -76,9 +75,8 @@ public class Notifications : Object {
 
         Notification notification = new Notification(_("Subscription request"));
         notification.set_body(jid.bare_jid.to_string());
-        try {
-            notification.set_icon(get_pixbuf_icon((new AvatarGenerator(40, 40)).draw_jid(stream_interactor, jid, account)));
```

Comment:
You should also remove the `get_pixbuf_icon()` method, which isn’t called anywhere anymore.

Reply:
@linkmauve I've seen this but wasn't sure, if this function could be useful in some later time.

Thanks for reviewing!